nncf/openvino/graph/metatypes/openvino_metatypes.py

```diff
@@ -403,6 +404,16 @@ class OVLogicalXorMetatype(OVOpMetatype):
     hw_config_names = [HWConfigOpName.LOGICALXOR]
 
 
+@OV_OPERATOR_METATYPES.register()
+class OVEmbeddingMetatype(OVOpMetatype):
```

Comment:
Why does `ONNXEmbeddingMetatype` have`weight_port_ids` but `const_channel_axis` does not exist for `OVEmbeddingMetatype`?

Reply:
For now, there is no configuration for weights in `cpu.json` for the `Embedding` type. This is why the `const_channel_axis` wouldn't have any effect. To be honest, `weight_port_ids` wouldn't have any effect too.
Should we remove or extend the current version, what you do think?